src/infra/main/template.yaml

```diff
@@ -200,15 +237,15 @@ Resources:
   TxMAIngressQueue:
     Type: AWS::SQS::Queue
     Properties:
-      QueueName: !Sub "${AWS::StackName}-TxMAIngressQueue"
+      QueueName: !Sub "${AWS::StackName}-TxMAIngressQueue" #test commit
       MessageRetentionPeriod: 1209600
       KmsMasterKeyId: !GetAtt SQSEncryptionKey.Arn
       RedriveAllowPolicy:
         redrivePermission: denyAll
       RedrivePolicy:
         deadLetterTargetArn: !GetAtt TxMAIngressDLQ.Arn
         maxReceiveCount: 3
```

Comment:
probabaly want to increase this 60 so that it retries every 5 sec for 5mins before giving up and going to dlq - allows us to handle a 5 min outage